src/d3a/models/area/__init__.py

```diff
@@ -148,6 +148,8 @@ def activate(self, bc=None):
         self.log.debug('Activating area')
         self.active = True
         self.dispatcher.broadcast_activate()
+        if self.redis_ext_conn is not None:
+            self.redis_ext_conn.sub_to_area_event()
```

Comment:
Just for understanding: You moved the subscribe into the activate method because there is no point listening to the channels before that, right?

Reply:
Explained offline. Thanks